common/badBrowserCheck.ts

```diff
@@ -0,0 +1,31 @@
+import Modernizr from 'modernizr';
+
+const features: (keyof typeof Modernizr)[] = ['flexbox', 'flexwrap', 'localstorage'];
+
+/**
+ * Checks for browser support in the specified feature. If the feature is not supported, the `BadBrowser` overlay will
+ * be shown.
+ *
+ * @param feature
+ * @return {boolean}
+ */
+const isSupported = (feature: keyof typeof Modernizr): boolean => {
+  if (!Modernizr[feature]) {
+    const element = document.getElementsByClassName('BadBrowser')[0];
+    element.className += ' is-open';
+
+    if (/iPhone|iPad|iPod|Android/i.test(navigator.userAgent)) {
+      element.className += ' is-mobile';
+    }
```

Comment:
Avoid nested conditionals
```
if (Mondernizr[feature]) return true;
const element = ....

```

Reply:
Have you looked into `Modernizr.addTest`? https://modernizr.com/docs/
It seems we could replace this check with a `ismobile` test

Reply:
AFAIK this is only for visual changes on the bad browser overlay. Does it really make sense to use Modernizr for that?

Reply:
I doubt it. We should be able to accomplish the same using responsive css